src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -128,46 +168,8 @@ def run_app_batch(
         run_id=None,
         dry_run=False,
     ):
-        try:
-            if params is None:
-                params = list()
-            return self._run_app_batch_internal(
-                app_id, params, tag, version, cell_id, run_id, dry_run
-            )
-        except Exception as e:
-            e_type = type(e).__name__
-            e_message = str(e).replace("<", "&lt;").replace(">", "&gt;")
-            e_trace = traceback.format_exc()
-            e_trace = e_trace.replace("<", "&lt;").replace(">", "&gt;")
-            e_code = getattr(e, "code", -1)
-            e_source = getattr(e, "source", "appmanager")
-            self._send_comm_message(
-                "run_status",
-                {
-                    "event": "error",
-                    "event_at": datetime.datetime.utcnow().isoformat() + "Z",
-                    "cell_id": cell_id,
-                    "run_id": run_id,
-                    "error_message": e_message,
-                    "error_type": e_type,
-                    "error_stacktrace": e_trace,
-                    "error_code": e_code,
-                    "error_source": e_source,
-                },
-            )
-            print(
-                "Error while trying to start your app (run_app_batch)!\n"
-                + "-----------------------------------------------------\n"
-                + str(e)
-                + "\n"
-                + "-----------------------------------------------------\n"
-                + e_trace
-            )
-            return
-
-    def _run_app_batch_internal(
-        self, app_id, params, tag, version, cell_id, run_id, dry_run
-    ):
+        if params is None:
+            params = list()
```

Comment:
This became part of the spec a while ago, where params are allowed to not exist, but are still expected to be an argument. It would make more sense for it to be a kwarg, but this keeps backward compatibility with existing app cells.